### PR TITLE
Update position of the overlay on opened

### LINKED ISSUE #524: Setting opened=true doesn't update the overlay position
When opening `vaadin-date-picker` by setting the property `opened=true` (instead of clicking the date-picker or calling `open()`), the overlay opens up in the top-left corner of the window. Apparently because `_updateAlignmentAndPosition()` gets never called like it does [here](https://github.com/vaadin/vaadin-date-picker/blob/13028b3d6dae928b7341c821f6b70a24faa687dd/src/vaadin-date-picker-mixin.html#L345).

### PR DESCRIPTION
**Pull request by YuriyVaadin**
Fixes #524

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-date-picker/525)

### CODE REVIEW

File: src/vaadin-date-picker-mixin.html [hunk 1]
```diff
@@ -399,6 +400,14 @@
         '';
     }
 
+    _openedChanged(opened) {
+      if (!opened) {
+        return;
+      }
+
+      this._updateAlignmentAndPosition();
```
Comment (jouni) [inline src/vaadin-date-picker-mixin.html]: Isn’t line 346 redundant (and even a harmful for performance) after this (calling the `_updateAlignmentAndPosition()` method inside the `open()` method?

Comment (YuriyVaadin) [reply src/vaadin-date-picker-mixin.html]: You are right! Thanks, I've removed that line.

**Comment by web-padawan** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/525)*

**Comment by web-padawan** [comment]
:lgtm:

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/525#-:-L82kvH5IiUcK1WJTdRG:bnfp4nl)*

**Comment by tomivirkki** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/525#-:-L82lJFozGptIry29fb2:bnfp4nl)*

**Pull request merged**